__fixtures__/test-project/web/package.json

```diff
@@ -23,6 +23,7 @@
     "react-dom": "18.2.0"
   },
   "devDependencies": {
+    "storybook": "7.0.12",
```

Comment:
I need this here to get tests to run correctly; we can remove it after a canary gets published